### PR TITLE
264 single category pages

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
 partial https://trello.com/c/LB8qXXSG/264-supporting-facts-actions-i-can-take-stage-iii-sub-pages

## Description

- Refactor routing
- Stub out a single category page with title

@neontribe/sea-map

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -35,19 +36,14 @@ type alias Model =
...
-
 init : () -> Url.Url -> Browser.Navigation.Key -> ( Model, Cmd Msg )
 init _ url key =
-    toUrl url
-        { key = key
-        , page = NotAlonePage NotAlone.Model
-        }
+    let
+        maybePage =
+            pageFromUrl url { key = key, page = NotAlonePage NotAlone.Model }
```
Comment (katjam) [inline src/Main.elm:43]: I'm not sure about this part in terms of passing that NotAlone.Model in.

File: src/Copy/Text.elm [hunk 2]
```diff
@@ -275,7 +279,7 @@ t key =
 
         -- Help Self Single Category1
         HelpSelfCategory1Title ->
-            ""
+            "COVID"
```
Comment (katjam) [inline src/Copy/Text.elm:282]: Hard coded for testing

File: src/Main.elm [hunk 3]
```diff
@@ -82,12 +78,17 @@ update msg model =
                     )
 
         UrlChanged url ->
-            toUrl url model
+            let
+                newPage =
+                    Maybe.withDefault (NotAlonePage {}) (pageFromUrl url model)
```
Comment (katjam) [inline src/Main.elm:83]: Here I pass an empty model instead of the Page.Model

File: src/Main.elm [hunk 4]
```diff
@@ -59,7 +55,7 @@ type Msg
     | PageLinkClicked Browser.UrlRequest
     | UrlChanged Url.Url
     | DefinitionMsg Definition.Msg
-    | HelpSelfMsg HelpSelf.Msg
+    | HelpSelfSingleMsg HelpSelfSingle.Msg
```
Comment (Nikomus) [inline src/Main.elm:58]: Does this need HelpSelfGridMsg HelpSelfGrid.Msg?

Comment (katjam) [reply src/Main.elm:58]: Right now the grid page doesn't have any Msg types

**Review by Nikomus — APPROVED** [review]
👍

**Pull request merged**